### PR TITLE
Java doc broken link fix and minor update

### PR DESCRIPTION
**Pull request by lupesko**
## Description ##
The Java on IntelliJ doc has a broken link that this commit fixes, and a couple of minor text edits.

### Changes ###
- Fix a broken link to the Java Infer API examples
- Text edits to the readme

### CODE REVIEW

**Comment by lupesko** [comment]
@lanking520 @andrewfayres please take a look.
@mxnet-label-bot add [pr-awaiting-review]

**Comment by andrewfayres** [comment]
Changes look good. Thanks for the link fix. CI seems to of failed and needs to be restarted.

**Comment by stu1130** [comment]
@mxnet-label-bot add [pr-awaiting-review]

File: docs/tutorials/java/mxnet_java_on_intellij.md [hunk 1]
```diff
@@ -167,5 +166,5 @@ java -cp target/javaMXNet-1.0-SNAPSHOT.jar:target/dependency/* mxnet.App
 For more information about MXNet Java resources, see the following:
 
 * [Java Inference API](https://mxnet.incubator.apache.org/api/java/infer.html)
-* [Java Inference Examples](https://github.com/apache/incubator-mxnet/tree/java-api/scala-package/examples/src/main/java/org/apache/mxnetexamples/infer/)
+* [Java Inference Examples](https://github.com/apache/incubator-mxnet/tree/java-api/scala-package/examples/src/main/java/org/apache/mxnetexamples/javaapi/infer)
```
Comment (lanking520) [inline docs/tutorials/java/mxnet_java_on_intellij.md]: Please change the URL to `master`

https://github.com/apache/incubator-mxnet/tree/master/scala-package/examples/src/main/java/org/apache/mxnetexamples/javaapi/infer

**Comment by andrewfayres** [comment]
Since this was a minor fix I went ahead and included it in #13405

**Comment by lupesko** [comment]
Resolved by @andrewfayres in #13405 - closing.

**Pull request closed without merging**